common/memcached/templates/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         app: {{ template "fullname" . }}
         component: memcached
       annotations:
-        {{- if and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested }}
+        {{- if and (and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested) $.Values.linkerd.enabled }}
```

Comment:
I think, you might also write it simply as `and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested $.Values.linkerd.enabled`

Reply:
Thought about the same, but then it's harder to find the exact condition for Stefan when he does mass changes, so I prefer keeping this as close to the expected original as possible.

Reply:
@majewsky what do you prefer ?

Reply:
I would leave it as it is. It's a minor detail, and we can change it later too. No reason to delay the tests for that.